hermes/templates/python/template.py.tpl

```diff
@@ -235,8 +235,8 @@ class DefaultSyntaxErrorHandler:
         return error
     def unexpected_eof(self):
         return self._error("Error: unexpected end of file")
-    def excess_tokens(self):
-        return self._error("Finished parsing without consuming all tokens.")
+    def excess_tokens(self, next_token):
+        return self._error("Finished parsing without consuming all tokens.  Next token was {next}".format(next=next_token))
```

Comment:
Ah, I didn't realize there were tests for hermes.  That's good to know.  But back to the root issue, I can make the error msg consistent across languages some time.